### PR TITLE
[DCOS-59730] [Diagnostic] Getting VERSION error during ./create_service_diagnostics_bundle.sh execution

### PR DESCRIPTION
**Pull request by mediapills**
move VERSION value back to create_service_diagnostics_bundle.sh

### CODE REVIEW

**Comment by mediapills** [comment]
Can not check do we have `.sh` files inside Docker image because of script on S3 doesn't work properly on Mac. The way with arguments acceptable and it was also in initial implementation of [e945987e17b2dee52af0a2ea69230bccb0fe26fe](https://github.com/mesosphere/dcos-sdk-service-diagnostics/commit/e945987e17b2dee52af0a2ea69230bccb0fe26fe)

**Comment by mediapills** [comment]
PTAL

File: python/create_service_diagnostics_bundle.sh [hunk 1]
```diff
@@ -110,4 +110,4 @@ container_run "rm -rf ${CONTAINER_DCOS_CLI_DIRECTORY} && mkdir ${CONTAINER_DCOS_
                  -exec cp --parents \{\} ${CONTAINER_DCOS_CLI_DIRECTORY} \;
                cd /
                dcos plugin add /tmp/dcos-core-cli-${DCOS_CLUSTER_MAJOR_MINOR_VERSION}.zip
-               ${CONTAINER_SCRIPT_PATH} ${*} --bundles-directory ${CONTAINER_BUNDLES_DIRECTORY}"
+               ${CONTAINER_SCRIPT_PATH} ${*} --bundles-directory ${CONTAINER_BUNDLES_DIRECTORY} --diagnostics-version ${VERSION}"
```
Comment (porridge) [inline python/create_service_diagnostics_bundle.sh]:
```suggestion
               ${CONTAINER_SCRIPT_PATH} ${*} \
                 --bundles-directory ${CONTAINER_BUNDLES_DIRECTORY} \
                 --diagnostics-version ${VERSION}"
```

File: python/create_service_diagnostics_bundle.sh [hunk 2]
```diff
@@ -5,7 +5,7 @@ set -eu -o pipefail
 readonly SCRIPT_DIRECTORY="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null && pwd)"
 readonly DCOS_SERVICE_DIAGNOSTICS_SCRIPT_PATH="dcos-sdk-service-diagnostics/python"
 
-readonly VERSION="$(head -1 "$SCRIPT_DIRECTORY/VERSION")"
+readonly VERSION='v0.4.0'
```
Comment (porridge) [inline python/create_service_diagnostics_bundle.sh]:
```suggestion
readonly VERSION='v0.5.0'
```

File: python/create_service_diagnostics_bundle.sh (hunk 1 above)
Comment (mediapills) [reply python/create_service_diagnostics_bundle.sh]: changed

File: python/create_service_diagnostics_bundle.sh (hunk 2 above)
Comment (mediapills) [reply python/create_service_diagnostics_bundle.sh]: updated

**Comment by porridge** [comment]
> while at it please also revert [this part](https://github.com/mesosphere/dcos-sdk-service-diagnostics/commit/fe6e8ef5dffa5dbac3f5a198253202ec36bc8404#diff-a2da7232dec2c4fc156ce8e4ae1a4228L44-L51) of my recent doc change.

How about this one?

**Comment by mediapills** [comment]
PTAL again please

**Pull request merged**